src/components/ThoughtAnnotationWrapper.tsx

```diff
@@ -13,41 +10,26 @@ import FauxCaret from './FauxCaret'
  */
 const ThoughtAnnotationWrapper: FC<
   PropsWithChildren<{
-    cursorOverlay?: boolean
+    annotationRef?: RefObject<HTMLDivElement | null>
     ellipsizedUrl?: boolean
     multiline?: boolean
     value?: string
     styleAnnotation?: React.CSSProperties
-    cssRaw?: SystemStyleObject
-    style?: React.CSSProperties
     isTableCol1?: boolean
-    textMarkup?: string
-    placeholder?: string
   }>
-> = ({
-  cursorOverlay,
-  ellipsizedUrl,
-  multiline,
-  value,
-  styleAnnotation,
-  cssRaw,
-  style,
-  children,
-  isTableCol1,
-  textMarkup,
-  placeholder,
-}) => {
+> = ({ annotationRef, ellipsizedUrl, multiline, value, styleAnnotation, children, isTableCol1 }) => {
   return (
     <div
       aria-label='thought-annotation'
+      ref={annotationRef}
       className={css({
         position: 'absolute',
         pointerEvents: 'none',
         userSelect: 'none',
         boxSizing: 'border-box',
-        width: '100%',
-        // maxWidth: '100%',
-        marginTop: '0',
+        lineHeight: multiline ? 1.25 : undefined,
+        marginTop: multiline ? 'calc(-0.12em - 0.5px) !important' : value ? '-0.425em' : undefined,
```

Comment:
Let's avoid the use of `!important`. There should be other ways to set up the styles or create Panda recipes to allow style overriding in a less brute force way.